### ISSUE TITLE
The right "way" to use/copy several routines to personal library (concerning copyright)

### ISSUE THREAD

**Issue by izsahara** [IDAES/idaes-pse#548]
Hi, im writing my own library (That would be distributed publicly on github) and im interested in using just two routines from `idaes.surrogate.pysmo.sampling`. Namely `SamplingMethods` and `LatinHypercubeSampling`. First off, would i be allowed to? if so would the following be correct in doing so

![image](https://user-images.githubusercontent.com/62043990/136647636-2eb093ac-a6f3-4bb7-a097-bd5dbd4b5a1e.png)

Thanks,

**Comment by eslickj** [comment]
Yes, you can use IDAES code you just have to comply with the license https://github.com/IDAES/idaes-pse/blob/main/LICENSE.md.  I think the typical way to do this would be to make IDAES a dependency of your project.  The advantage of going to dependency route is that you don't need to maintain a fork of the code. 

@ksbeattie can probably give you better information on this.

**Comment by adowling2** [comment]
Alternately, you may find this function helpful: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.qmc.LatinHypercube.html
